wp-content/themes/illdy-child/layout/js/translations_page.js

```diff
@@ -193,6 +222,69 @@ var TranslationsPage = (function(window, $) {
       return container;
     };
 
+    this.filterLangs = function(list, filter, resource_filter, translationsData) {
+        var searchText = filter.value.toUpperCase();
+        var items = list.getElementsByTagName('li');
+        for (i = 0; i < items.length; i++) {

```

Comment:
You might be able to use Array.forEach for these but the for loop works just fine.